tests/pages/contribute.py

```diff
@@ -0,0 +1,39 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as expected
+from selenium.webdriver.support.ui import WebDriverWait as Wait
+
+from .base import BasePage
+
+
+class ContributePage(BasePage):
+
+    _url = '{base_url}/{locale}/contribute'
+
+    _video_play_locator = (By.CSS_SELECTOR, '.video-play')
+    _video_close_locator = (By.ID, 'modal-close')
+    _video_modal_locator = (By.ID, 'modal')
+
+    _event_link_locator = (By.CSS_SELECTOR, '.extra-event .event-link')
+
+    def open_video_modal(self):
```

Comment:
Suggestion: A more descriptive name might be `play_video`. The video could even be a `PageRegion` with an `is_displayed` property and `close` function.